Sources/EpoxyCore/SwiftUI/LayoutUtilities/MeasuringUIViewRepresentable.swift

```diff
@@ -12,12 +12,12 @@ import SwiftUI
 /// `sizeThatFits(…)` method.
 ///
 /// - SeeAlso: ``SwiftUIMeasurementContainer``
-public protocol MeasuringUIViewRepresentable: UIViewRepresentable
+public protocol MeasuringUIViewRepresentable: UIViewRepresentableOrNSViewRepresentable
```

Comment:
maybe this is too big of a change, but can we change the name of this protocol to `MeasuringViewRepresentable` to remove the `UIView` part?

Reply:
That sounds good to me. Agreed that it felt weird for this to be named "UIView" but be usable with NSViews.

Reply:
What do you think about this, given it would be a breaking change, @brynbodayle?